src/ComponentParser.ts

```diff
@@ -303,15 +305,36 @@ export default class ComponentParser {
           }
         }
 
-        if (node.type === "ExportNamedDeclaration" && node.declaration != null) {
+        if (node.type === 'VariableDeclaration') {
+          this.vars.add(node);
```

Comment:
Type cast to appease the [TS error](https://github.com/carbon-design-system/sveld/runs/4684825310?check_suite_focus=true#step:3:14).

```suggestion
          this.vars.add(node as unknown as VariableDeclaration);
```

Reply:
Do we need the `as unknown` part? Isn't `VariableDeclaration` a subtype of `Node`, which would make it a simple down-cast?